service/resource/persistentvolume/error.go

```diff
@@ -24,3 +24,17 @@ var missingAnnotationError = microerror.New("missing annotation")
 func IsMissingAnnotationError(err error) bool {
 	return microerror.Cause(err) == missingAnnotationError
 }
+
+var waitResourceError = microerror.New("waitResource")
+
+// IsWaitTimeout asserts invalidConfigError.
+func IsWaitReosurce(err error) bool {
+	return microerror.Cause(err) == waitResourceError
+}
+
+var waitTimeoutError = microerror.New("waitTimeout")
```

Comment:
It should be
```
 var waitResourceError = microerror.New("wait resource")
```
```
 var waitTimeoutError = microerror.New("wait timeout")
```
But I would just create
```
var timeoutError = microerror.New("timeout")
```
And use it in both cases with additional annotation.